hibernate-core/src/main/java/org/hibernate/cfg/AvailableSettings.java

```diff
@@ -1511,4 +1511,15 @@
 	 */
 	String ALLOW_JTA_TRANSACTION_ACCESS = "hibernate.jta.allowTransactionAccess";
 
+	/**
+	 *  Setting that allows to perform update operations outside a transaction boundary.
+	 *
+	 *  After the consolidation of JPA support into hibernate-core, Hibernate conforms with
```

Comment:
Rather than explain about the consolidation, I'd stick to the point:
 "since version 5.2 Hibernate conforms with .."